### PR TITLE
Remove AccountApplicationsPage from testsuite

### LINKED ISSUE #15198: Remove AccountApplicationsPage from testsuite
### Description

Tests that are not testing the account console should not relay on it. This task is to remove [AccountApplicationsPage](https://github.com/keycloak/keycloak/blob/bfce612641a70e106b20b136431f0e4046b5c37f/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/AccountApplicationsPage.java) and usage of it, re-writing to use REST APIs as applicable.

### PR DESCRIPTION
**Pull request by wojnarfilip**
Closes #15198

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OfflineServletsAdapterTest.java [hunk 1]
```diff
@@ -1,34 +1,30 @@
 package org.keycloak.testsuite.adapter.servlet;
 
 import javax.ws.rs.core.UriBuilder;
-import java.util.List;
+import java.util.*;
```
Comment (vmuzikar) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OfflineServletsAdapterTest.java:4]: Please don't use star imports.

Comment (wojnarfilip) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OfflineServletsAdapterTest.java]: Thanks @vmuzikar I removed the asterisk imports and fixed the flaky test the PR previously introduced. Could you take a look now please ?

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/util/AccountHelper.java [hunk 2]
```diff
@@ -32,7 +34,7 @@ public static boolean updatePassword(RealmResource realm, String username, Strin
         }
 
         UserRepresentation userRepresentation = userResult.get();
```
Comment (miquelsi) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/util/AccountHelper.java:36]: All these lines above are no longer necessary, they are executed in `getUserId` method.

Comment (wojnarfilip) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/util/AccountHelper.java]: Thanks for noting this, it's removed.

**Review by miquelsi — DISMISSED** [review]
Great job @wojnarfilip! LGTM

**Comment by wojnarfilip** [comment]
@stianst @pedroigor Could you retrigger Keycloak-CI please ? The failures in WebAuthnIT shouldn't be related to this PR.

**Review by miquelsi — DISMISSED** [review]

**Comment by jonkoops** [comment]
@wojnarfilip did a rebase, that should re-run the latest of the test suite.

**Review by mposolda — APPROVED** [review]

**Pull request merged**